components/mediation-admin/org.wso2.carbon.mediation.library/src/main/java/org/wso2/carbon/mediation/library/service/MediationLibraryAdminService.java

```diff
@@ -148,7 +148,7 @@ public LibraryInfo getLibraryInfo(String libName, String packageName) throws Axi
 		OMElement impEl = SynapseImportSerializer.serializeImport(synImport);
 		if (impEl != null) {
 			try {
-				SynapseImport synapseImport = configuration.getSynapseImports().get("{"+packageName+"}"+libName);
+				SynapseImport synapseImport = configuration.getSynapseImports().get("{" + packageName + "}" + libName);
```

Comment:
Since whole class(MediationLibraryAdminService.java) has this issue, I formatted whole class in this PR: https://github.com/wso2/carbon-mediation/pull/454